source/common/upstream/outlier_detection_impl.cc

```diff
@@ -52,6 +52,7 @@ void DetectorHostMonitorImpl::eject(MonotonicTime ejection_time) {
   ASSERT(!host_.lock()->healthFlagGet(Host::HealthFlag::FAILED_OUTLIER_CHECK));
   host_.lock()->healthFlagSet(Host::HealthFlag::FAILED_OUTLIER_CHECK);
   num_ejections_++;
+  eject_time_backoff_++;
```

Comment:
Should this be capped at some max value? During prolonged outages this would likely get really high right?

Reply:
Correct. Thanks for pointing this out. I added `max_ejection_time` config to cap it.